tools/generator/src/DTO/Platform.swift

```diff
@@ -19,14 +19,16 @@ struct Platform: Equatable, Hashable, Decodable {
     struct OSVersion: RawRepresentable, Hashable, Decodable {
         let rawValue: String
         let fullVersion: String
+        let semanticVersion: SemanticVersion?
```

Comment:
Because of how `fullVersion` is constructed, thus should always be non-nil. We should surface an error if it's not.

Reply:
Then, why do we keep `rawValue` around?

Reply:
Because that is the user-facing value set, so we show that in disambiguated target names.

Reply:
Mainly it was because it comes through as `15.0` instead of `15.0.0` and I wanted to show that, but we can have logic to do the same thing with `SemanticVersion` printing.

Reply:
I have made it required.